web_archive_date/readme/ROADMAP.rst

```diff
@@ -0,0 +1,3 @@
+Format of `archive_date` is not the same as `write_date` shown in the metadata. Values are formatted in javascript. Can't inherit the method and adapt the value there.
+
+TO BE DONE: Check how to format the value in the QWeb view somehow.
```

Comment:
```suggestion
```
You can remove the file. Thank you @JasminSForgeFlow for the Javascript changes! 👍🏽

Reply:
Updated, Thanks